test/test_structured_1d.jl

```diff
@@ -32,6 +32,13 @@ isdir(outdir) && rm(outdir, recursive=true)
       atol = 1.0e-5)
   end
 
+  @trixi_testset "elixir_euler_sedov_hll_Davis.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_sedov_hll_Davis.jl"),
+      l2   = [1.278661029299215, 0.0663853410742763, 0.9585741943783386],
+      linf = [3.1661064228547255, 0.16256363944708607, 2.667676158812806],
+      tspan = (0.0, 12.5))
+  end
```

Comment:
Did you check the runtime of this new test for CI?

Reply:
In [this CI](https://github.com/trixi-framework/Trixi.jl/actions/runs/5465343805/jobs/9948653677) we have

` [ Info: Testset elixir_euler_sedov_hll_Davis.jl finished in 11.826439 seconds.`
(both with and without coverage)